packages/ses/src/get-source-url.js

```diff
@@ -9,7 +9,7 @@ const sourceMetaEntryRegExp =
 // On account of the mechanics of regular expressions, scanning from the end
 // does not allow us to capture every pair, so getSourceURL must capture and
 // trim until there are no matching comments.
-const sourceMetaEntriesRegExp = new RegExp(
+const sourceMetaEntriesRegExp = new FERAL_REG_EXP(
```

Comment:
I don't love it but not worth fixing. I do have a non-serious suggestion for your amusement:
```suggestion
const sourceMetaEntriesRegExp = new /./.constructor(
```